spec/brainstem/query_strategies/paginator_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "spec_helper"
+
+describe Brainstem::QueryStrategies::Paginator do
+  using_mysql = ActiveRecord::Base.connection.instance_values["config"][:adapter] =~ /mysql/i
+  if using_mysql
+    xit '- Tests requiring sqlite have not been run.' do
```

Comment:
Do we want the `xit`? Isn't that what the if-else is for?

Reply:
This is to give feedback to whoever is running the tests that there is a whole branch of tests that have been missed. I uncovered code that had failing tests for some time but that branch of tests had not been run before being merged.